Source/WebCore/css/StyleProperties.cpp

```diff
@@ -576,12 +577,34 @@ String StyleProperties::fontValue() const
     if (auto shorthand = isSingleFontShorthand())
         return getValueNameAtomString(shorthand.value());
 
+    // Font stretch values can only be serialized as keywords.
+    // While it's not clear the CSS font specification calls for this, web platform tests expect us to convert percentages to keywords.
+    CSSValueID fontStretchPercentageAsKeyword = CSSValueInvalid;
+    if (int fontStretchPropertyIndex = findPropertyIndex(CSSPropertyFontStretch); fontStretchPropertyIndex != -1) {
```

Comment:
Is this a good place to use `notFound` instead of `-1`? (not sure what our guideline is here, especially since this is an `int` and not `size_t`)

Reply:
This function uses `-1`, and not `notFound`. It’s private to just this source file, and we can change it to use `std::optional` at some point to clean things up.